celery_migration_app.py

```diff
@@ -19,17 +21,21 @@
     'result_serializer': 'json',
     'accept_content': ['json']})
 
+r = redis.Redis(host='0.0.0.0', port=6379, db=0, decode_responses=True)
 
-@app.task
-def migrate_photo(photo_title, photo_url, album_title):
+
+@app.task(autoretry_for=(RequestException, HttpError),
```

Comment:
retry tasks for api failures.  Can run with concurrency > 1 and tasks will do exponential backoff if being throttled.